db/migrate/20150107044854_users_without_location_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class UsersWithoutLocationCleanup < ActiveRecord::Migration
+  def up
+    User.delete_all(location: [nil,''])
```

Comment:
Hello,

I would rather implement find_in_batches inside a task instead of a migration:

http://apidock.com/rails/ActiveRecord/Batches/find_in_batches

Also with find_in_batches you can define the size of the batch to process.

Cheers,

Juan.